### ISSUE TITLE
Solar + wind's build cost and expected output should reflect local weather forecast

### ISSUE THREAD

**Issue by toddmedema** [toddmedema/electrify#103]
It's currently based on a single hardcoded capacity factor, obviously not correct

1. DONE: Update solar output calculation in updateSupplyFacilitiesFinances in Game.tsx to account for lat/long (add lat/long to LOCATIONS in constants) AND month of year...  
2. Build facilities menu should reflect full output calculation (incl weather and lat long) in average output and $/MWh for wind + solar

For simplicity in determining "average output", can use the weather for the full upcoming year as a model year, e.g. don't need to simulate/forecast every year of service life uniquely

**Comment by toddmedema** [comment]
Perhaps "capacity factor" is the best way to estimate this

Thought: How much would I expect the climate to meaningfully change in a way that would impact solar/wind capacity factor? Uhhh, without getting into complex climate change modeling, minimally... so, then, I could argue that capacity factor could just be locally-calculated once

Which would also work well since that particular logic of generating facilities isn't used much - only in BuildFacilities and InitGame, so it's ok if it works a bit to actually forecast out.

So, basically, expand the `export function GENERATORS` in Facilities.tsx to take in (location? weather?) and calculate custom capacity factors for solar and wind — and then, of course, verify that the updates flow through and make sense.

For the record, existing values on master when starting SF 2020 scenario, 200MW:
wind: 62MW/year, $27.4/mwh
solar: 44MW/year, $25.1/mwh
